dev-lang/nim/nim-1.6.10-r1.ebuild

```diff
@@ -0,0 +1,223 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit bash-completion-r1 edo multiprocessing toolchain-funcs xdg-utils
+
+PATCH_PV="1.6.6_p1"
```

Comment:
Style nitpick: put this above `inherit`s.

Reply:
Done